examples/tools/scene_viewer.rs

```diff
@@ -1,3 +1,8 @@
+//! A simple way to view glTF models with Bevy.
+//! Just run `cargo run --release --example scene_viewer -- /path/to/model.gltf#Scene0`,
```

Comment:
I don't think the --  is necessary

Reply:
You are correct. Just tested this, and - at least with 1.60 on windows - `cargo run` seems to pass additional paramters to the target binary. 
I'll remove all ` -- ` from `cargo run`s in the examples.